applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -1639,6 +1639,276 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
       #XPING: average 0.650
       OK
 
+Testing GPS AT commands
+-----------------------
+
+Before you test the GPS AT commands, check the following configuration values:
+
+* :option:`CONFIG_SUPL_CLIENT_LIB` - must be enabled to use the :ref:`supl_client`
+* :option:`CONFIG_SLM_SUPL_SERVER` - use the default ("supl.google.com") or specify a different server
+* :option:`CONFIG_SLM_SUPL_PORT` - use the default (7276) or specify a different port
+
+Complete the following steps to test the functionality provided by the :ref:`SLM_AT_GPS`:
+
+1. Test standalone GPS:
+
+   a. Enable support for GNSS in the modem.
+
+      .. parsed-literal::
+         :class: highlight
+
+         **AT+CFUN=0**
+         OK
+
+         **AT%XSYSTEMMODE=0,0,1,0**
+         OK
+
+   .. modem_config_start
+
+   b. Configure the modem.
+      The required commands depend on the version of the nRF9160 DK.
+
+      For nRF9160 DK v0.9.0:
```

Comment:
Is this section correct?
The first command is needed for v0.9.0, and the second for all versions <= v0.10.x?
@junqingzou @lats1980

Reply:
Yes, due to change of GPS_EN signal (check schematic of different versions)
"<= v0.9.0", use the MAGPIO pin to control LNA and filter
">= v0.10.x", use the COEX0 pin to control LNA, filter and switch
Latest DK comes with on-board GPS antenna.
To use onboard antenna,  AT%XCOEX0=1,1,1565,1586
To use an external antenna, AT%XCOEX0      // this will disable LNA as it's expected to be inside the external antenna

Reply:
Updated, please check again.